keps/sig-node/20190131-runas-groupid.md

```diff
@@ -245,14 +245,20 @@ There are other potentially unresolved discussions in that PR which need a follo
 
 ## Graduation Criteria
 
+Beta
 - Publish Test Results from Master Branch of Cri-o To http://prow.k8s.io [#72253](https://github.com/kubernetes/kubernetes/issues/72253)
 - Containerd and CRI-O tests included in k/k CI [#72287](https://github.com/kubernetes/kubernetes/issues/72287)
 - Make CRI tests failures as release informing
```

Comment:
were these steps done? the linked issued aged out without apparent PRs/updates referenced